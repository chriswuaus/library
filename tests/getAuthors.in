LIST authors
add book test.csv 111111
list authors
add collection test3nodup.csv
list authors
add collection test.csv
list authors
EXit
