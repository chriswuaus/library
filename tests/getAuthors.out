user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Harper Lee

user: 9 books successfully added.

user: Bob Kane
Douglas Adams
Harper Lee
Homer
J.K. Rowling
J.R.R. Tolkien
Jamie Lee Curtis
T.S. Eliot
Virgil
W.E. Johns

user: 7 books successfully added.

user: Bob Kane
Douglas Adams
Harper Lee
Homer
J.K. Rowling
J.R.R. Tolkien
Jamie Lee Curtis
T.S. Eliot
Virgil
W.E. Johns

user: Ending Library process.
